src/sidebar/services/groups.js

```diff
@@ -141,6 +146,7 @@ function groups(
     if (isSidebar) {
       uri = getDocumentUriForGroupSearch();
     }
+    let directLinkedGroup = settings.group;
```

Comment:
It doesn't look like this var is re-assigned later, in which case it should be declared with `const`.